applications/DEMApplication/custom_constitutive/DEM_KDEM_with_damage_parallel_bond_CL.cpp

```diff
@@ -326,8 +326,8 @@ namespace Kratos {
 
         if (mDebugPrintingOption) {
 
-            const int sphere_id = element1->GetProperties().GetValue(DEBUG_PRINTING_ID_1);
-            const int neigh_sphere_id = element2->GetProperties().GetValue(DEBUG_PRINTING_ID_2);
+            long unsigned int sphere_id = element1->GetProperties().GetValue(DEBUG_PRINTING_ID_1);
```

Comment:
I did not see the CI. What was the problem with `const int`?

Reply:
We were comparing element1->Id() and sphere_id. Element1->Id() is of long unsigned int type so the CI complained that they had different types.